eth/_utils/numeric.py

```diff
@@ -125,3 +125,25 @@ def integer_squareroot(value: int) -> int:
         x = y
         y = (x + value // x) // 2
     return x
+
+
+def bitwise_xor(a: Hash32, b: Hash32) -> Hash32:
+    """
+    Return the xor of hash ``a`` and hash ``b``
+    """
+    if len(a) != 32:
+        raise ValueError(
+            "Arguments must be of size 32: Got: {0}".format(
+                len(a)
+            )
+        )
+    if len(b) != 32:
+        raise ValueError(
+            "Arguments must be of size 32: Got: {0}".format(
+                len(b)
+            )
+        )
+    c = b''
+    for i in range(32):
+        c += bytes([a[i] ^ b[i]])
+    return Hash32(c)
```

Comment:
I think this should be updated to:

```python
result = bytes(bit_a, bit_b for bit_a, bit_b in zip(a, b)
return Hash32(result)
```

Also can you update the `a, b` variables to be `left, right` to avoid single character names.

Reply:
@pipermerriam oh yes that's better! 👍 

nitpick:
```python
result = bytes(bit_a ^ bit_b for bit_a, bit_b in zip(a, b))
return Hash32(result)
```